src/nimibook/books.nim

```diff
@@ -0,0 +1,68 @@
+import std / os
+import nimibook /  [types, entries]
+import jsony
+
+# fix for jsony: quote in string. https://github.com/treeform/jsony/issues/14
+proc dumpHook*(s: var string, v: string) =
+  when nimvm:
+    s.add '"'
+    for c in v:
+      case c:
```

Comment:
Can this be simplified with the use of : 

* [addEscaped](https://nim-lang.github.io/Nim/system.html#addEscapedChar%2Cstring%2Cchar)
* [addQuoted](https://nim-lang.github.io/Nim/system.html#addQuoted%2Cstring%2CT)
* [escape](https://nim-lang.github.io/Nim/strutils.html#escape%2Cstring%2Cstring%2Cstring) / unescape

I didn't analyzed it in details, but it seems like we could simplify it ?

Reply:
this and the following I think it should be better addressed in [this PR to jsony](https://github.com/treeform/jsony/pull/15). I ran into the issue while dumping `Book` object, found the fix and directly applied it. When jsony merges the fix we should remove this and update jsony requirement.